flowauth/frontend/src/RoleList.jsx

```diff
@@ -14,5 +14,11 @@ const styles = (theme) => ({
     paddingBottom: theme.spacing.unit * 2,
   },
 });
-const GroupList = Lister(GroupDetails, "Groups", getGroups, deleteGroup);
-export default withStyles(styles)(GroupList);
+const RoleList = Lister(
```

Comment:
I think it would be good to display which server each role corresponds to (e.g. by having a heading per server with roles for that server listed underneath, or a server-name label alongside each role).

Reply:
Overall I think I'd prefer to have a separate header per server, but if a different option is quicker to implement then that's fine too.